lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -77,6 +77,17 @@ def is_enabled(cls, course, user=None):
         return bool(user and has_access(user, 'staff', course, course.id))
 
 
+def show_analytics_dashboard_message(course_key):
+    """
+    Defines whether or not the analytics dashboard URL should be displayed.
+
+    Arguments:
+        course_key (CourseLocator): The course locator to display the analytics dashboard message on.
+    """
+    return settings.ANALYTICS_DASHBOARD_URL and (not hasattr(course_key, 'ccx')
+                                                 or settings.FEATURES.get('ENABLE_CCX_ANALYTICS_DASHBOARD_URL', False))
```

Comment:
I agree, xors are never terribly readable. :/ I will fix this up to make it easier to understand.